### PR TITLE
Update the GUI testing page

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
Updated note about supported QT version and changed sub-heading.

### CODE REVIEW

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 1]
```diff
@@ -100,7 +102,7 @@ NOTE: If you are testing against the oCIS server, set *OCIS=true* in the *config
 
 NOTE: If there are problems with starting Squish, please check the log file `~/.squish/squishlibraryx.log.x`. Also make sure that the `~/squish-for-qt-6.x.x/etc/paths.ini` is user-readable.
 
-===== Extra Steps for Infinite Scale
+===== Extra Steps for Infinite Scale or ownCloud server with oauth2
```
Comment (mmattel) [inline modules/ROOT/pages/appendices/guitest.adoc:105]:
```suggestion
===== Extra Steps for Infinite Scale or ownCloud Server with Oauth2
```

**Comment by mmattel** [comment]
Needs backporting to 5.0

**Review by mmattel — APPROVED** [review]

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 2]
```diff
@@ -51,7 +51,9 @@ After building the ownCloud Desktop App, install and configure Squish. To instal
 . Download the latest version of Squish from {squish-url}[froglogic] to a location of your choice.
 . Depending upon the version and system you are using, you will get a file like `squish-x.x.x-qtxxx-x64.run`.
 
-NOTE: For desktop client >= 2.11, Squish with QT 5.15 is required.
+NOTE: For desktop client >= 2.11 and <5, Squish with QT 5.15 is required.
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/guitest.adoc:54]:
```suggestion
NOTE: For Desktop Client >= 2.11 and <5, Squish with QT 5.15 is required.
```

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 3]
```diff
@@ -51,7 +51,9 @@ After building the ownCloud Desktop App, install and configure Squish. To instal
...
+
+NOTE: For desktop client >=5, Squish with QT 6.2 is the minimal supported QT version.
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/guitest.adoc:56]:
```suggestion
NOTE: For Desktop Client >=5, Squish with at least QT 6.2 is required.
```

**Review by EParzefall — APPROVED** [review]

**Pull request merged**